dali/python/nvidia/dali/_autograph/impl/api.py

```diff
@@ -130,40 +129,47 @@ def _attach_error_metadata(e, f):
     e.ag_error_metadata = _ErrorMetadata(cause_tb, metadata, message, source_map, __file__)
 
 
-# class StackTraceMapper(tf_stack.StackTraceMapper):
-#   """Remaps generated code to code it originated from."""
+class StackTraceMapper(tf_stack.StackTraceMapper):
+    """Remaps generated code to code it originated from."""
 
-#   def __init__(self, converted_fn):
-#     super().__init__()
-#     self._source_map = converted_fn.ag_source_map
-#     # This may be called repeatedly: once on entry, by the superclass, then by
-#     # each child context manager.
-#     self._cached_map = None
+    def __init__(self, converted_fn):
+        super().__init__()
+        self._source_map = converted_fn.ag_source_map
+        # This may be called repeatedly: once on entry, by the superclass, then by
+        # each child context manager.
```

Comment:
I don't understand this comment nor do I know what "call" it refers to - it sits between two assignments (no calls).

Reply:
This comment is inherited from TensorFlow.
The whole structure of StackTraceMappers/Filters is used as context managers, and this comment is a bit exaggerated in the expectation of further inheritance from this class.